packages/contracts/src/contracts/current/protocol/Exchange/MixinBatchOperations.sol

```diff
@@ -0,0 +1,143 @@
+/*
+
+  Copyright 2017 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.4.19;
+
+import './mixins/MExchangeCore.sol';
+import "../../utils/SafeMath/SafeMath.sol";
+
+/// @dev Consumes MExchangeCore
+contract MixinBatchOperations is
```

Comment:
Is there a better name than `BatchOperations`? `fillOrKillOrder` isn't a batch function, and there will also be `fillOrderNoThrow` later on.